tests/platform_tests/api/test_sfp.py

```diff
@@ -718,6 +718,11 @@ def test_reset(self, request, duthosts, enum_rand_one_per_hwsku_hostname, localh
                 continue
             info_dict = port_index_to_info_dict[sfp_port_idx]
 
+            # when an optic XCVR goes through sfp reset on an Arista platform it goes into lpmode
```

Comment:
@arista-nwolfe Agree that on Arista platform if we reset the optics, the module will remain in low power mode unless Xcvrd reinitializes the optics to bring it out of low power. So, Why on your platform the interface is not able to reinitialize by doing no shut (see the test case completely)?

Reply:
Sounds like there are 2 bugs that I'll fix in a separate review: